### ISSUE TITLE
Setup Proper Analytics

### ISSUE THREAD

**Issue by AlexeyMK** [AlexeyMK/alexeymk.com#5]
Currently, the analytics stack is limited to
- GA (so broken because of GA4)
- Heap (can delete)
- Clearbit (keep).

The simplest move is probably to move to Google Tag Manager (set up, give Alexey full ownership, and move stuff there).  

Aside: Some page speed concerns - open to being pitched a server-side tracking alternative, but not willing to have a monthly cost for that (IE, it can't require needing an actual server).  Is there a simple solution? GTM seems to think we need Docker or Node: https://developers.google.com/tag-platform/tag-manager/server-side/manual-setup-guide

**Comment by mesinkasir** [comment]
This script usually burdens your site's performance. 
So the solution is to use Google Meta HTML, later we can check the analysis in Google Search Console. 😇

**Comment by AlexeyMK** [comment]
Yes - we discussed this.  Let's just suck it up and add GTM instead. I insist.

**Comment by Blaumaus** [comment]
You may try [Swetrix](https://swetrix.com) instead, I'm building this project. It's a well-maintained, open-source and GDPR compliant Google Analytics alternative.

We also support [server-side tracking](https://docs.swetrix.com/events-api) or in case you'd use the frontend script instead - it's very lightweight (around 4KB)
